services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/HankeServiceITests.kt

```diff
@@ -999,6 +999,28 @@ class HankeServiceITests : DatabaseTest() {
         assertThat(hanke.perustaja?.email).isEqualTo("teppo@example.test")
     }
 
+    @Test
+    fun `generateHankeWithApplication sets hanke name according to limit and saves successfully`() {
+        val expectedName = "a".repeat(MAXIMUM_HANKE_NIMI_LENGTH)
+        val tooLongName = expectedName.plus("aaa")
```

Comment:
If this test ever fails, it might be handy if the extra part is different from the expected part.
```suggestion
        val tooLongName = expectedName.plus("bbb")
```

Reply:
The `plus()` can be used as an operator:
```suggestion
        val tooLongName = expectedName + "bbb"
```

Reply:
Both fixed as suggested.